src/remat.jl

```diff
@@ -232,12 +232,12 @@ end
 *(adjA::Adjoint{T,<:ReMat{T}}, B::ReMat{T}) where {T} = adjA.parent.adjA * sparse(B)
 *(adjA::Adjoint{T,<:FeMat{T}}, B::ReMat{T}) where {T} =
     mul!(Matrix{T}(undef, rank(adjA.parent), size(B, 2)), adjA, B)
-
+#=
 LinearAlgebra.mul!(C::AbstractMatrix{T}, adjA::Adjoint{T,<:FeMat{T}},
-        B::ReMat{T}) where {T} = mulαβ!(C, adjA, B)
-
-function mulαβ!(C::Matrix{T}, adjA::Adjoint{T,<:FeMat{T}}, B::ReMat{T,1},
-        α=true, β=false) where {T}
+        B::ReMat{T}) where {T} = mul!(C, adjA, B)
+=#
```

Comment:
do we need to preserve the code commented out here?

Reply:
No, I should have removed it